packages/cli/src/commands/dev.js

```diff
@@ -19,6 +19,12 @@ export const builder = (yargs) => {
       description: 'Which dev server(s) to start',
       type: 'array',
     })
+    .positional('webpackConfig', {
+      alias: 'wpc',
+      description:
+        'Optional string of Webpack DevServer config options, e.g. `--wpc="--port=1234"`',
+      type: 'string',
+    })
```

Comment:
Ooh, I love this... This is what I've wanted for a long time. 

Could we make it more generic? Cause I would like to make this also work for jest, lint, etc.

Maybe something like `--forward`. In yarn and npm this used to just be `--`: `yarn test -- --watchAll`